### PR TITLE
Website/conf: Add Partners section to display media, community partner logos 

### LINKED ISSUE #1449: Display GraphQLConf Media or Community Partner logos
### Description

Add a Partners section below Sponsors on the GraphQLConf site to display Media and Conference Partners logos.

### Motivation

The is a promised benefit to media and community orgs that help promote GraphQLConf on the sites or meetup pages with our provided [banner images](https://github.com/graphql/marketing/tree/main/assets/GraphQLConf).

### Collaboration

Meena will deliver a PR for this.

### PR DESCRIPTION
**Pull request by meenakshi-dhanani**
Closes #1449 

## Description
- Add partners section below sponsors in the footer
- Deep link for partners section - graphql.org/conf/#partners
- Display logos for community and media partners

Screenshot 2023-05-18 at 2 54 26 PM

### CODE REVIEW

**Comment by Urigo** [comment]
@meenakshi-dhanani can you look why the build failed?

**Comment by meenakshi-dhanani** [comment]
> @meenakshi-dhanani can you look why the build failed?

Yeah I'm not sure. I added a few images for logos, and I have no issues locally.

File: src/components/Conf/Partners/index.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/* eslint-disable tailwindcss/no-custom-classname */
+import React from "react"
+import { ReactComponent as GraphQLWeekly } from "../../../../static/img/conf/Partners/GraphQLWeekly.svg"
+import { ReactComponent as GraphQLWTF } from "../../../../static/img/conf/Partners/GraphQLwtf.svg"
```
Comment (TuvalSimha) [inline src/components/Conf/Partners/index.tsx:4]:
```
import { ReactComponent as GraphQLWTF } from "../../../../static/img/conf/Partners/Graphqlwtf.svg"
```

Replace `GraphQLwtf` to `Graphqlwtf`

**Review by TuvalSimha — CHANGES_REQUESTED** [review]

File: src/components/Conf/Partners/index.tsx (hunk 1 above)
Comment (meenakshi-dhanani) [reply src/components/Conf/Partners/index.tsx:4]: Thank you, it worked. Another set of eyes definitely helps!

**Comment by Urigo** [comment]
thank you both!

**Pull request merged**